components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -73,6 +73,14 @@ export class DashboardView extends PolymerElement {
                                 ' Kubernetes Engine',
                         link: `${DOCS}/getting-started-gke/`,
                     },
+                    {
+                        text: 'EKS for Kubeflow',
+                        desc: 'Get Kubeflow running on AWS. ' +
+                        'This guide is a quickstart to' +
+                        ' deploying Kubeflow on Amazon Elastic' +
+                        ' Container Service for Kubernetes (Amazon EKS)',
+                        link: `${DOCS}/getting-started-aws/`,
+                    },
```

Comment:
@prodonjs Should we make this conditional based on our platform detection checks?

Reply:
This seems consistent with the other links here which provide guidance on how to setup KF on various platforms so I think it's good to add statically.

@kmarquardsen If you are interested in doing so in a subsequent PR, we have an area for exposing cloud-provider specific links and it would be great to have someone contribute useful links for AWS-based deployments.

Reply:
@prodonjs I did notice that. Happy to do the platform specific links in another PR.

Reply:
Awesome, if you are interested you can take a look at what we did for GCP in https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/public/components/resources/cloud-platform-data.js.

You'll want to add an icon for AWS and a function that returns an object with a shape similar to the one returned for GCP with `links`, `title`, `logo`, and `name` properties.

Reply:
Sounds great @prodonjs I'll create an issue and assign it to myself.